_includes/scenarios/integrate/create-attribute-for-peer.md

```diff
@@ -1,15 +1,237 @@
+
+| Attribute Type | Attribute Owner | Possible? | Automation      | Examples/Reason                                                                                                                                                                                                                                                          |
+| -------------- | --------------- | :-------: | --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Identity       | Sender          |     ✗     | `N/A`           | Use [ShareAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#shareattributerequestitem) instead.                                                                                                                                                    |
+| Identity       | Recipient       |     ✓     | `USER_DECISION` | University sends student his certificate (Propose would be inappropriate in this case, because the student should not be able to return his own value).                                                                                                                  |
+| Identity       | `<empty>`       |     ✓     | `USER_DECISION` | An empty owner defaults to an Attribute with `owner=<Recipient>`. This is needed for Requests inside of RelationshipTemplates, since you don’t know the enmeshed Address of your peer before the Relationship is established.                                            |
+| Relationship   | Sender          |     ✓     | `AUTO_ACCEPT`   | Company sends new customer his customer number.                                                                                                                                                                                                                          |
+| Relationship   | Recipient       |     ✓     | `USER_DECISION` | With this combination the **Sender asks the Recipient for the one-time permission** to write a RelationshipAttribute once **and** the **Sender defined a value** which can either be accepted and stored, or rejected. Thus, the user cannot change the value by itself. |
+| Relationship   | `<empty>`       |     ✓     | `USER_DECISION` | An empty owner defaults to an Attribute with `owner=<Recipient>`. This is needed for Requests inside of RelationshipTemplates, since you don’t know the enmeshed Address of your peer before the Relationship is established.                                            |
```

Comment:
Should we also mention, that it is not possible to have a third party as owner (which I hope is implemented this way)?

Reply:
I just saw that apparently it is not implemented. We should fix this (and other bugs @britsta found) quicky.